docs/ref/contrib/postgres/indexes.txt

```diff
@@ -78,3 +78,24 @@ available from the ``django.contrib.postgres.indexes`` module.
 
     .. _buffering build: https://www.postgresql.org/docs/current/static/gist-implementation.html#GIST-BUFFERING-BUILD
     .. _fillfactor: https://www.postgresql.org/docs/current/static/sql-createindex.html#SQL-CREATEINDEX-STORAGE-PARAMETERS
+
+``HashIndex``
+=============
+
+.. class:: HashIndex(fillfactor=None, **options)
+
+    .. versionadded:: 2.1
+
+    Creates a `hash index
+    <https://www.postgresql.org/docs/current/static/hash.html>`_.
```

Comment:
This page doesn't exist and after a quick search, I didn't see a suitable replacement.

Reply:
I was a bit presumptuous here. Will have a look for an alternative link or remove.

Reply:
There only seem to be specific sections for brin, gin, gist and spgist: https://www.postgresql.org/docs/current/static/internals.html